unit/goto-cc/armcc_cmdline.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Diffblue Limited. All Rights Reserved.
```

Comment:
I think the licence header is wrong - thats the internal Diffblue header - CBMC contributions require a different header.